src/node_internals.h

```diff
@@ -102,46 +102,46 @@ struct sockaddr;
-    V(config)                                                                 \
-    V(contextify)                                                             \
-    V(domain)                                                                 \
-    V(fs)                                                                     \
-    V(fs_event_wrap)                                                          \
-    V(heap_utils)                                                             \
-    V(http2)                                                                  \
-    V(http_parser)                                                            \
-    V(inspector)                                                              \
-    V(js_stream)                                                              \
-    V(messaging)                                                              \
-    V(module_wrap)                                                            \
-    V(options)                                                                \
-    V(os)                                                                     \
-    V(performance)                                                            \
-    V(pipe_wrap)                                                              \
-    V(process_wrap)                                                           \
-    V(serdes)                                                                 \
-    V(signal_wrap)                                                            \
-    V(spawn_sync)                                                             \
-    V(stream_pipe)                                                            \
-    V(stream_wrap)                                                            \
-    V(string_decoder)                                                         \
-    V(symbols)                                                                \
-    V(tcp_wrap)                                                               \
-    V(timers)                                                                 \
-    V(trace_events)                                                           \
-    V(tty_wrap)                                                               \
-    V(types)                                                                  \
-    V(udp_wrap)                                                               \
-    V(url)                                                                    \
-    V(util)                                                                   \
-    V(uv)                                                                     \
-    V(v8)                                                                     \
-    V(worker)                                                                 \
-    V(zlib)
+#define NODE_BUILTIN_STANDARD_MODULES(V)                                       \
```

Comment:
it's going to continue being a problem every time we run clang-format until someone lets it be merged. I say leave it.

Reply:
@refack This is done by `make format-cpp`, so the next time someone touches this bit it will be reindented again - I'd prefer we just go with whatever `make format-cpp` gives and forget about formatting issues, that'd be easier.

Reply:
That's my preference, the it will be fixed when there's a real change to the list.
But it's just a preference, non-blocking.